apps/volunteer/shifts/shop.py

```diff
@@ -0,0 +1,13 @@
+from ..event_date import edt
+
+shop_shifts = {
+    "Shop Helper": {
+        "Shop": [
```

Comment:
Might need an extra few shifts Thursday for merch.

Reply:
I see conversation in shop channel that they don't need volunteers for Thurs.